docs/api/flink/Function.md

```diff
@@ -152,6 +152,34 @@ Input: `POLYGON ((-0.5 -0.5, -0.5 0.5, 0.5 0.5, 0.5 -0.5, -0.5 -0.5))`
 
 Output: `POLYGON ((-0.5 -0.5, 0.5 -0.5, 0.5 0.5, -0.5 0.5, -0.5 -0.5))`
 
+## ST_PointN
```

Comment:
Please add docs for ST_ExteriorRing too.

Reply:
Added